commands/status_test.go

```diff
@@ -57,6 +58,7 @@ PMM Server:
 PMM Client:
 	Connected        : false
 	pmm-admin version: 
```

Comment:
let's not leave pmm-admin version empty in this test. I know that it's almost imposible case but, let's set unknown in case it's empty.

Reply:
Changed to "unknown".